lib/acts_as_list/active_record/acts/list.rb

```diff
@@ -174,8 +174,12 @@ def move_lower
           return unless lower_item
 
           acts_as_list_class.transaction do
-            lower_item.decrement_position
-            increment_position
+            if lower_item.send(position_column) != self.send(position_column)
```

Comment:
Just wondering with these methods. Do we need the branching here. Could the first branch not handle all situations (i.e. consecutive numbering and non-consecutive?)